tests/expressions/tests.py

```diff
@@ -802,6 +802,20 @@ def test_boolean_expression_combined(self):
             [self.example_inc.ceo, self.max],
         )
 
+    def test_aggregation_groupby_expression_wrapper(self):
```

Comment:
```suggestion
    def def test_q_expression_annotation_with_aggregation(self):
```

Please move this test to the `annotations/tests.py` where we have tests for related regressions, see 8a6df55f2dd5131282084a4edfd48f63fbf8c69a and aeb8996a6706cad3e96d8221760c1cb408ee7ed9.

Reply:
Also, this test will crash on Oracle, we should use:
```python
@skipUnlessDBFeature('supports_boolean_expr_in_select_clause')
```

Reply:
We currently have a test with name `test_q_expression_annotation_with_aggregation` in `annotations/tests.py`.
So, I moved the test to `annotations/tests.py` and renamed it to `test_grouping_by_expression_wrapper_with_q_object`